WordPressAuthenticator/Signin/LoginEmailViewController.swift

```diff
@@ -533,3 +558,22 @@ extension LoginEmailViewController: GIDSignInDelegate {
         signInGoogleAccount(signIn, didSignInFor: user, withError: error)
     }
 }
+
+// MARK: - AppleAuthenticatorDelegate
+
+extension LoginEmailViewController: AppleAuthenticatorDelegate {
+
+    func showWPComLogin(loginFields: LoginFields) {
+        self.loginFields = loginFields
+         performSegue(withIdentifier: .showWPComLogin, sender: self)
+    }
```

Comment:
I think this should say `// no-op`? Because in this context: coming from Jetpack Remote Install > Sign up > Continue with Apple, it would never call the `showWPComLogin()` method...right?

Reply:
I do believe `showWPComLogin` is needed. It's nothing to do with Jetpack at this point, it's all about the SIWA flow.

It pertains to this logic in `AppleAuthenticator`:

```
guard !existingNonSocialAccount else {

    if existing2faAccount {
        self?.show2FA()
        return
    }

    self?.updateLoginEmail(wpcomUsername)
    self?.logInInstead()
    return
}
```

Basically, what happens here is if an Apple ID is used where the email has already been used on a WP account, the flow will re-direct to WP login.

Reply:
ohhh. Nice!